thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -1309,7 +1308,7 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
                         joinGroup->setAtMostLimitHit(); // also clears existing rows
                         break;
                     }
-                    KLBlobProviderAdapter adapter(keyManager, &activity.contextLogger);
+                    KLBlobProviderAdapter adapter(keyManager, nullptr);
```

Comment:
it looks like the remote side (slavmain) would return track and serialize blob stats.
Please test an example to verify same whether using local or remote. Regression test 'indexblobs.ecl' may work as a test query.

Reply:
Blob stats are not being tracked for keyed join.  However, the blob stats for keyed join was never correct because slavmain was not tracking blob stats.  Also, hthor was not tracking blob stats.  It would be better to address this in a separate jira as the changes affect not only thor  but roxie and hthor too. TODO: create ticket once jira is working again.

Reply:
jira: https://track.hpccsystems.com/browse/HPCC-31002